### ISSUE TITLE
Surveys: What to do with sent survey after deletion?

### ISSUE THREAD

**Issue by paulbert** [open-learning-exchange/planet#3479]
If we send a survey to a member, then delete that survey, the survey remains in the members "mySurveys" list.  The member can still take that survey successfully and the results will show up in the submissions list.

I'm leaning towards not changing this since it doesn't break the experience at all, but we could make this a low priority issue if others disagree

**Comment by yuviii** [comment]
@paulbert Agreed with your thoughts. However i guess might be we need to add survey valid date for the case if the survey is deleted because it is not longer needed any more and old.

**Comment by paulbert** [comment]
Ok so I'll make this low priority since it's not broken, just not ideal.

When a survey gets deleted, we should remove all pending submissions for that survey so we keep the data for anyone who already took it, but we don't collect any more data from people it was sent out to.
